src/crystal/system/win32/time.cr

```diff
@@ -53,21 +53,20 @@ module Crystal::System::Time
     ((filetime.dwHighDateTime.to_u64 << 32) | filetime.dwLowDateTime.to_u64).to_f64 / FILETIME_TICKS_PER_SECOND.to_f64
   end
 
-  @@performance_frequency : Int64 = begin
-    ret = LibC.QueryPerformanceFrequency(out frequency)
-    if ret == 0
-      raise RuntimeError.from_winerror("QueryPerformanceFrequency")
-    end
+  @@performance_frequency : Int64?
 
-    frequency
+  private def self.performance_frequency
+    @@performance_frequency ||= begin
+      LibC.QueryPerformanceFrequency(out frequency)
+      frequency
+    end
```

Comment:
**polish:** Maybe this could be a bit more concise with `class_getter`?

```suggestion
class_getter performance_frequency : Int64 do
  LibC.QueryPerformanceFrequency(out frequency)
  frequency
```